hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/token/TokenGetInfoUsage.java

```diff
@@ -4,15 +4,15 @@
 import static com.hedera.node.app.hapi.fees.usage.token.entities.TokenEntitySizes.TOKEN_ENTITY_SIZES;
 import static com.hedera.node.app.hapi.utils.fee.FeeBuilder.BASIC_ENTITY_ID_SIZE;
 
+import com.hedera.hapi.node.base.Key;
+import com.hedera.hapi.node.transaction.Query;
 import com.hedera.node.app.hapi.fees.usage.QueryUsage;
 import com.hedera.node.app.hapi.utils.fee.FeeBuilder;
-import com.hederahashgraph.api.proto.java.Key;
-import com.hederahashgraph.api.proto.java.Query;
 import java.util.Optional;
 
 public class TokenGetInfoUsage extends QueryUsage {
     private TokenGetInfoUsage(final Query query) {
-        super(query.getTokenGetInfo().getHeader().getResponseType());
+        super(query.tokenGetInfo().header().responseType());
```

Comment:
`query.tokenGetInfo()` is `null`.

In the old implementation, it returned a default value and therefore did not throw (although that also looks incorrect).

There are more places like this that would need adjustment.